### ISSUE TITLE
[FR] Support moving application to SD card

### ISSUE THREAD

**Issue by cozyplanes** [devRant/devRant#1]
Please give official support moving devRant to sdcard in android.
As far as i know, it may be different in appcelerator, but it is just a line of code in `AndroidManifest.xml`. 

_Sent from my Lge LG-F460L using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by DFoxinator** [comment]
Thanks, definitely makes sense!

And congrats on first issue submitted :)

**Comment by cozyplanes** [comment]
@DFoxinator i kinda advertised this repo in devRant, but not getting upvotes 😭😭😭 
You really should post a rant and see if people dont like posts that looks like i stole something from yoh.

_Sent from my Lge LG-F460L using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Bikonja** [comment]
@DFoxinator @cozyplanes I agree, I first saw a rant by someone else (neither of you two) and I thought why is this guy passing off something unofficial as official and wanted to report him, but then I realised it actually is official :)

As for the original issue, as @cozyplanes said, yes, it is just one line, click [here](https://developer.android.com/guide/topics/data/install-location.html) for Android docs. Don't know about Appcelerator, though.

**Comment by cozyplanes** [comment]
@Bikonja _hey! That `wintron` account is me.... Yeah, super sorry for_ @dfox _, but i am using it for testing api stuffs, so please take no offense_ 

_Sent from my Lge LG-F460L using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by DFoxinator** [comment]
I'm going to be announcing this officially but purposely haven't yet. No need to post about it on the app yet.

**Comment by cozyplanes** [comment]
@DFoxinator ok, but why? Are you preparing for an update that trogus mentioned few days ago?

_Sent from my Lge LG-F460L using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by DFoxinator** [comment]
@cozyplanes no, it just takes some time for me to write up an announcement, and I'm also still making some tweaks to this.

**Comment by DFoxinator** [comment]
I changed the title of this because we'll be using the tag to designate feature requests. Makes it easier to read the issues I think.

**Comment by LoganS1** [comment]
I found a link that may help with this @DFoxinator \:
http://primegap.net/2011/01/03/appcelerator-titanium-quick-tip-enable-move-to-sd-in-android-applications/
